### ISSUE TITLE
Use panel based progress bar instead of tqdm if available?

### ISSUE THREAD

**Issue by jsignell** [intake/intake#291]
Once the new GUI is implemented in panel (#225) rather than ipywidgets - the progress bar will be the last use of ipywidgets. It would be nice to provide an alternate/replacement progress bar that is implemented in panel so that ipywidgets will no longer be needed at all.

**Comment by martindurant** [comment]
Agreed - however tqdm is also used in text mode. We should be able to simply turn off its ipywidgets interface, but I don't know if we want to choose between the two very different systems. Perhaps we just drop it in text more, or provide extremely simple feedback for that case ("working...")

**Comment by martindurant** [comment]
@jsignell , do you intend to work on this? I would be happy to bin tqdm with ipywidgets and just use text mode, or use a panel progress widget, whichever is easier.

**Comment by jsignell** [comment]
I have no immediate plans to work on this. I think a text version would probably be easiest.

**Comment by philippjfr** [comment]
I'd love to have a progress widget in panel, but it doesn't exist currently.
